src/dex/uniswap-v2/config.ts

```diff
@@ -0,0 +1,452 @@
+import { DexParams } from './types';
+import { DexConfigMap } from '../../types';
+import { Network, SwapSide } from '../../constants';
+
+export const UniswapV2Config: DexConfigMap<DexParams> = {
+  UniswapV2: {
+    [Network.MAINNET]: {
+      subgraphURL: 'https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v2',
+      factoryAddress: '0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f',
+      initCode:
+        '0x96e8ac4277198ff8b6f785478aa9a39f403cb768dd02cbee326c3e7da348845f',
+      poolGasCost: 80 * 1000,
+      feeCode: 30,
+    },
+    [Network.ROPSTEN]: {
+      factoryAddress: '0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f',
+      initCode:
+        '0x96e8ac4277198ff8b6f785478aa9a39f403cb768dd02cbee326c3e7da348845f',
+      poolGasCost: 80 * 1000,
+      feeCode: 30,
+    },
+  },
+  ApeSwap: {
+    [Network.BSC]: {
+      subgraphURL:
+        'https://graph.apeswap.finance/subgraphs/name/ape-swap/apeswap-subgraph',
+      factoryAddress: '0x0841bd0b734e4f5853f0dd8d7ea041c241fb0da6',
+      initCode:
+        '0xf4ccce374816856d11f00e4069e7cada164065686fbef53c6167a63ec2fd8c5b',
+      poolGasCost: 100 * 1000,
+      feeCode: 20,
+    },
+    [Network.POLYGON]: {
+      subgraphURL:
+        'https://api.thegraph.com/subgraphs/name/apeswapfinance/dex-polygon',
+      factoryAddress: '0xcf083be4164828f00cae704ec15a36d711491284',
+      initCode:
+        '0x511f0f358fe530cda0859ec20becf391718fdf5a329be02f4c95361f3d6a42d8',
+      poolGasCost: 100 * 1000,
+      feeCode: 20,
+    },
+  },
+  BaguetteSwap: {
+    [Network.AVALANCHE]: {
+      subgraphURL:
+        'https://api.thegraph.com/subgraphs/name/baguette-exchange/baguette',
+      factoryAddress: '0x0841bd0b734e4f5853f0dd8d7ea041c241fb0da6',
+      initCode:
+        '0xf4ccce374816856d11f00e4069e7cada164065686fbef53c6167a63ec2fd8c5b',
+      poolGasCost: 80 * 1000,
+      feeCode: 30,
+    },
+  },
+  BakerySwap: {
+    [Network.BSC]: {
+      subgraphURL: 'https://api.bscgraph.org/subgraphs/name/bakeryswap',
+      factoryAddress: '0x01bF7C66c6BD861915CdaaE475042d3c4BaE16A7',
+      initCode:
+        '0xe2e87433120e32c4738a7d8f3271f3d872cbe16241d67537139158d90bac61d3',
+      feeCode: 30,
+      router: '0x4B8107Bbdf7f7adF7B42a1C9cF14FAD8d32111C0',
+      adapters: {
+        [SwapSide.SELL]: [
+          {
+            name: 'BscAdapter01',
+            index: 11,
+          },
+        ],
+        [SwapSide.BUY]: [],
```

Comment:
I think it's not acceptable to have an empty array, `getAdapters` should return `null` if there are no adapters for a DEX, so can have `null` here.

Reply:
fixed